plugins/orchestrator-backend/dist-dynamic/package.json

```diff
@@ -50,20 +50,18 @@
     "export-dynamic": "janus-cli package export-dynamic-plugin"
   },
   "dependencies": {
-    "@octokit/rest": "^19.0.3",
-    "@severlessworkflow/sdk-typescript": "^3.0.3",
     "@urql/core": "^4.1.4",
     "cloudevents": "^8.0.0",
     "express": "^4.18.2",
     "express-promise-router": "^4.1.1",
     "fs-extra": "^10.1.0",
     "json-schema": "^0.4.0",
     "openapi-backend": "^5.10.5",
-    "openapi-types": "^12.1.3",
     "winston": "^3.11.0",
     "yn": "^5.0.0",
     "moment": "^2.29.4",
     "ajv-formats": "^2.1.1",
+    "@severlessworkflow/sdk-typescript": "^3.0.3",
```

Comment:
Should be at the top. `yarn install` sorts dependenciecies alphabetically

Reply:
This file is auto-generated